### DISCUSSION TITLE
怎样在PCL自定义主页插入访问接口后返回显示文本

### DISCUSSION THREAD

**Discussion by zzr-shift** [Meloong-Git/PCL#4624] [提问] [answered]
额 纯小白一个 真的不知怎么做 就类似于回声洞那种点一下换一条 我想在自定义主页里面插入一言 谢谢帮忙

**Comment by MoYuan-CN** [comment]
点一下换一条？
同时实现文本插入和点击切换，使用
```xaml
<local:MyTextButton Text="{cave}" EventType="刷新主页" />
```
插入在你想要的位置

**Reply by WForst-Breeze** [reply]
Author 指的应该是允许通过外部 API 提供文本，然后在 PCL 主页实现类似于回声洞的效果（即每刷新一次文字改变一次）

**Reply by MoYuan-CN** [reply, marked as answer]
那貌似没法了，`{cave}` 不支持外部自定义

**Reply by Joker2184** [reply]
> 那貌似没法了，`{cave}` 不支持外部自定义

有办法了（

在线构建 + 脚本 每次构建随机插入一条自定义语句（））

**Comment by Joker2184** [comment]
魔改HINT.JPG
